src/components/VContentPage.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+  <div class="pt-5 md:pt-10" dir="ltr">
```

Comment:
What is the reason for setting direction here? It is probably breaking 'About' page for RTL.

Reply:
Good question: all of those pages had `dir="ltr"` set so I was just preserving the current behavior. I'll test if it should be removed.

Reply:
In discussion with WP i18n folks we decided to set the dir on these pages because they're largely untranslated and also not RTL compatible to begin with. If they're translated _and_ they've been fixed to support RTL then it can be removed.

Reply:
Ah, that's right. Thanks Sara! I'll test the status of translations _and_ how they look in RTL.

Reply:
What do you think is the best time to flip the switch is? When any RTL language translation is available? Or when all have been translated to some percentage? I guess it's not possible to do it per-language...

Reply:
I think we could do per language as long as translation percentage is available at run time (which I think the banner uses, so it must be, right?)

Reply:
@obulat  I made a new issue for this. None of these RTL languages have any translations, still. 

https://github.com/WordPress/openverse-frontend/issues/1118